StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory.swift

```diff
@@ -502,7 +503,7 @@ extension PaymentSheetFormFactory {
                 return
             }
 
-            phoneElement.selectedCountryCode = countryCode
+            phoneElement.setSelectedCountryCode(countryCode, shouldUpdateDefaultNumber: true)
```

Comment:
I'm not seeing anywhere else we set this property, so shouldn't we clean up the custom setter now that we're using a function instead?

Reply:
Whenever I create a function to set a property with additional parameters, I like having the setter to perform the default flow so it can still be assigned using `=`. That said, since this is an internal api (`@_spi` protection for the class) and we are not using it, it makes sense to remove it, if it was a public API I would keep it.

Reply:
I kinda question whether we should have so much logic in a setter though? A function flags more to the reader that more could be happening under the hood, whereas if I'm setting something, I'm not usually expecting it to have any custom validation (and therefore failure modes)